src/couch_mrview/src/couch_mrview_http.erl

```diff
@@ -530,9 +551,20 @@ parse_param(Key, Val, Args, IsDecoded) ->
             JsonKey = ?JSON_DECODE(Val),
             Args#mrargs{start_key = JsonKey, end_key = JsonKey};
         "keys" when IsDecoded ->
-            Args#mrargs{keys = Val};
+            case Val of
+                [Val1] ->
+                    Args#mrargs{start_key = Val1, end_key = Val1};
```

Comment:
If we intend for this case to be equivalent to key=... we should also reset (set to default) `keys=...`